eoverdracht/leveranciersspecificatie.md

```diff
@@ -250,19 +250,21 @@ Met het endpoint:
 }
 ```
 
-Dit laatste endpoint dient het `[base]` path te zien waarop notificaties ontvangen kunnen worden middels de `Task` resource. De volgende paragraaf beschrijft het notificatie mechanisme. Het betreft dus een notificatie dat er een nieuwe \(of gewijzigde\) `Task` resource klaar staat bij verzendende partij.
+Dit laatste endpoint dient het `[base]` path te zijn waarop notificaties ontvangen kunnen worden middels de `Task` resource. De volgende paragraaf beschrijft het notificatie mechanisme. Het betreft dus een notificatie dat er een nieuwe \(of gewijzigde\) `Task` resource klaar staat bij verzendende partij. Het path dient zonder `/` geregistreerd te worden.
```

Comment:
> Het path dient zonder `/` geregistreerd te worden.

Wordt hiermee bedoelt met een postfix `/`, dus m.a.w. niet eindigend op `/`? Dat is niet helemaal duidelijk.